src/main/java/edu/kis/powp/jobs2d/drivers/DriverComposite.java

```diff
@@ -25,4 +27,8 @@ public void setPosition(int x, int y) {
             driver.setPosition(x, y);
         }
     }
+
+    public List<Job2dDriver> getDrivers() {
+        return drivers;
```

Comment:
It is not a safe approach. You should at least return a copy of the list (full deep copy would be best).